### PR TITLE
CATD-401-a Sample App log analytic event changed body not sent

### PR DESCRIPTION
**Pull request by You-Hsuan**
Fixed: Changes to the bottom field on this view are not reflected in logged analytic events, where they should be.

### CODE REVIEW

File: SampleApp/SampleApp/Screens/EventsLoggingView.swift [hunk 1]
```diff
@@ -9,33 +9,61 @@
...
+            switch response {
+            case .success(let isLogged):
+                self?.result = "Success with result isLogged: \(isLogged)"
+            case .failure(let error):
+                self?.result = "Error with: \(error.localizedDescription)"
+            }
+        }
+    }
+
+    func setupParameters(userId: String, provider: String) -> [String: Any] {
```
Comment (paleksandrs) [inline SampleApp/SampleApp/Screens/EventsLoggingView.swift:38]: maybe this could be private?

**Review by paleksandrs — APPROVED** [review]

File: SampleApp/SampleApp/Screens/EventsLoggingView.swift (hunk 1 above)
Comment (You-Hsuan) [reply SampleApp/SampleApp/Screens/EventsLoggingView.swift]: Yes, I missed this, thanks!

**Comment by rossrossp** [comment]
I'm seeing some layout constraint warnings:
image

**Comment by rossrossp** [comment]
Nothing is happening when I log an event. I get no logging result and nothing comes through on Charles:

https://user-images.githubusercontent.com/6325131/163192110-9d9bed71-3538-4971-8233-c39001ed053a.mov

**Comment by You-Hsuan** [comment]
> I'm seeing some layout constraint warnings: 

I was using "iPhone 11 Pro" simulator for developing and I didn't see the constraint warnings.
But when I switch to iPod touch, I can see the warnings you mentioned.
After running on more simulators, I found that this warnings appear on smaller device only.
The warnings say those conflicts come from the constraint between TextField and keyboard. 

From other's experience, it looks like it's a bug from Apple.
On UIKit, we can hide the warnings by setting `.autocorrectionType = .no` , but I haven't found the solution for this on SwiftUI yet.
Some references: [link 1](https://github.com/hackiftekhar/IQKeyboardManager/issues/1616#issuecomment-595981792), [link 2](https://codecat15.medium.com/nslayoutconstraint-tuisysteminputassistantview-error-in-ios-13-iphone-se-simulator-7cec8dcc98d8), [link 3](https://developer.apple.com/forums/thread/668821)

Also, there's other warning in the console, but we can just ignore it based on Apple's response: 
https://developer.apple.com/forums/thread/691361

**Comment by You-Hsuan** [comment]
> Nothing is happening when I log an event. I get no logging result and nothing comes through on Charles:

Hi @rossrossp I'm not able to reproduce it, as I can see the call has been triggered on my side.
Could you confirm if your device has been registered successfully, and the requests have valid token please?
Also, if the response: "success" is false, app will wait for another 45s to send the next request from the queue.
Or could we pair review this on a call please? Thank you!

**Comment by You-Hsuan** [comment]
> > I'm seeing some layout constraint warnings:
> 
> I was using "iPhone 11 Pro" simulator for developing and I didn't see the constraint warnings. But when I switch to iPod touch, I can see the warnings you mentioned. After running on more simulators, I found that this warnings appear on smaller device only. The warnings say those conflicts come from the constraint between TextField and keyboard.
> 
> From other's experience, it looks like it's a bug from Apple. On UIKit, we can hide the warnings by setting `.autocorrectionType = .no` , but I haven't found the solution for this on SwiftUI yet. ( [link 1](https://github.com/hackiftekhar/IQKeyboardManager/issues/1616#issuecomment-595981792), [link 2](https://codecat15.medium.com/nslayoutconstraint-tuisysteminputassistantview-error-in-ios-13-iphone-se-simulator-7cec8dcc98d8), [link 3](https://developer.apple.com/forums/thread/668821) )
> 
> Also, there's other warning in the console, but we can just ignore it based on Apple's response: https://developer.apple.com/forums/thread/691361

**[Update]**
I finally find the solution for it! 
After I wrapped VStack inside a ScrollView, I don't see these constraints warnings anymore.
I'll also fix the same issue on `Identity Control` page. Thank you!

**Comment by You-Hsuan** [comment]
> @You-Hsuan I'm really sorry but I just tested again and only saw one analytic event tracked, despite 3 being loaded and me having clicked on 2 of them. Shall we take a look together tomorrow? Please put something in my calendar if you want to look together!

No problem, I've scheduled a meeting. Thank you!

**Review by rossrossp — APPROVED** [review]

**Pull request merged**